CMakeLists.txt

```diff
@@ -243,7 +244,7 @@ install(
         hazelcast/generated-sources/src/
         ${CMAKE_CURRENT_BINARY_DIR}/include/
 
-    TYPE INCLUDE
```

Comment:
I could not understand what is this change for. They seem to be the same directory default but letting it be determined automatically sounds more portable.

Reply:
Updated the description. See item 1.